omero/sysadmins/index.rst

```diff
@@ -43,9 +43,51 @@ series of commands.
     unix/server-install-homebrew
     unix/server-binary-repository
     unix/server-postgresql
-    unix/install-web
     advanced-install
 
+********************
+OMERO.web Deployment
+********************
+
+OMERO.web is a Python client of the OMERO platform that provides a
+web-based UI and JSON API.
+This section provides guidance on how to install and set up OMERO.web on
+any of the supported UNIX and UNIX-like platforms. Specific walkthroughs are
+provided for several systems, with detailed step-by-step instructions.
+OMERO.web can be either deployed with the OMERO.server or **separately** from 
+the OMERO.server. Deploying separately is **recommended**.
+
+Reading through the :doc:`unix/install-web/web-deployment` first
+is recommended.
+
+.. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    unix/install-web/web-deployment
+    unix/install-web/walkthrough/omeroweb-install-centos7-ice3.6
+    unix/install-web/walkthrough/omeroweb-install-debian-ice3.6
+    unix/install-web/walkthrough/omeroweb-install-ubuntu-ice3.6
+    unix/install-web/walkthrough/omeroweb-install-with-server-centos7-ice3.6
+    unix/install-web/walkthrough/omeroweb-install-with-server-debian-ice3.6
+    unix/install-web/walkthrough/omeroweb-install-with-server-ubuntu-ice3.6
+
+
+***************************************
+OMERO.web Maintenance and Customization
+***************************************
+
+This section contains instructions for administering and customizing your OMERO.web installation.
+
+.. toctree::
+    :maxdepth: 1
+    :titlesonly:
+
+    unix/install-web
+    customization
+    omeroweb-upgrade
```

Comment:
I think it would be fine to have this below the original server upgrade page, not sure it really makes sense to have it up here when you presumably need to upgrade your server first

Reply:
with us moving towards full decoupling you could argue that we should be able to upgrade web only

Reply:
Does this content actually allow you to do that though? What about the database? (sorry if that's a stupid question!)

Reply:
changes are made to a config file it is not part of the DB